Update KeyConcepts.md

BaoBaoGitHub opened the pull request:
According to offical architecture document, the data flow should be raw data->tool layer tables->domain layer tables.

As a result, roles in  pipelines shoue be collectors->extractors->converters->enrichers.

Among them, the function of extractors should be 
- extract data from `raw data table` to **`tool layer tables`**
rather than 
- extract data from `raw data table` to **`domain layer tables`**.

BaoBaoGitHub committed:
Update KeyConcepts.md

According to offical architecture document, the data flow should be raw data->tool layer tables->domain layer tables.
As a result, roles in  pipelines shoue be collectors->extractors->converters->enrichers.
Among them, the function of extractors should be extract data from raw data table to **tool layer tables** rather than extract data from raw data table to **domain layer tables**.

BaoBaoGitHub committed:
Merge pull request #1 from BaoBaoGitHub/BaoBaoGitHub-UpdateKeyConcepts

Update KeyConcepts.md

BaoBaoGitHub committed:
Update ToolLayerSchema.md

Naming convension of tool layer schema tables should be  `_tool_{plugin}_{entity}` rather than `_raw_{plugin}_{entity}`.

Startrekzky reviewed (commented):
Good catch. Can you also update the current version of docs?

BaoBaoGitHub committed:
Update KeyConcepts.md

The function of extractors is extracting data from `raw data tables` to `tool layer tables`.

BaoBaoGitHub committed:
Update ToolLayerSchema.md

Naming convension of tool layer schema tables should be `_tool_{plugin}_{entity}` rather than  `_raw_{plugin}_{entity}`.

BaoBaoGitHub committed:
Update KeyConcepts.md

The function of `Extractors` is extracting data from  `raw data tables` to `tool layer tables`.

Startrekzky reviewed (approved):
LGTM
